chapter_common_operators/batch_norm.md

```diff
@@ -32,7 +32,7 @@ The code snippet to define `batch_norm` is as follows.
 
 ```{.python .input  n=2}
 # Save to the d2ltvm package.
-import topi
+import topi # from tvm import topi # as of v0.7.0
```

Comment:
We will do an overhaul of the code to be compatible with the latest version of TVM in the near future. For now, as in the [beginning of the tutorial](https://github.com/d2l-ai/d2l-tvm/blob/master/chapter_getting_started/install.md) we specified the the code runs on `tvm-0.7-dev1`, we want to make it consistent throughout the tutorial.

Reply:
> Thanks for the PR. Please see my comments inline.

Thank you very much for `d2ltvm` I found it very helpful, and your material is complementary to the tvm tutorial!

Reply:
> We will do an overhaul of the code to be compatible with the latest version of TVM in the near future. For now, as in the [beginning of the tutorial](https://github.com/d2l-ai/d2l-tvm/blob/master/chapter_getting_started/install.md) we specified the the code runs on `tvm-0.7-dev1`, we want to make it consistent throughout the tutorial.

Sounds good. I was just thinking of helping other reader since it took some time to figure this out.
The prebuilt binary was not working for my system, so I built myself using the latest `tvm`; hence, I ran into the the changes.